zerberus/zerberusgui.cpp

```diff
@@ -210,21 +221,22 @@ void ZerberusGui::updateButtons()
 void ZerberusGui::onSoundFontLoaded()
       {
       bool loaded = _futureWatcher.result();
+      bool wasNotCanceled = !_progressDialog->wasCanceled();
       _progressTimer->stop();
       _progressDialog->reset();
-      if (!loaded) {
-            QMessageBox::warning(this,
-            tr("MuseScore"),
-            tr("Cannot load SoundFont %1").arg(_loadedSfPath));
-            }
-      else {
+      if (loaded) {
             QListWidgetItem* item = new QListWidgetItem;
             item->setText(_loadedSfName);
             item->setData(Qt::UserRole, _loadedSfPath);
             files->insertItem(0, item);
+            emit valueChanged();
+            emit sfChanged();
```

Comment:
these only need to be emitted when actually succeed in loading. (In fact FluidGui::soundFontAddClicked does not emit these if its soundfont load fails.)  So that's why I moved emits here...also makes it a little cleaner.